lib/huginn_acumen_order_agent/acumen_order_agent.rb

```diff
@@ -111,33 +115,37 @@ def handle(event)
                 'endpoint' => endpoint,
             }
             client = AcumenOrderClient.new(faraday, auth)
-
+            data = event.payload
             order_codes = event.payload['order_codes']
+            new_event = interpolated['output_mode'].to_s == 'merge' ? data.dup : {}
```

Comment:
Pretty sure this will throw an error if `output_mode` doesn't exist.

Reply:
No it don't, I tested it.